odkx-src/sync-endpoint-cloud-setup.rst

```diff
@@ -360,10 +364,26 @@ Connecting to your virtual machine
 
   .. code-block:: console
 
-    $ docker stack ls
+    $ sudo docker stack ls
 
-  If there are 8 (or 7 without https) services running under the name
-  `syncldap`, everything is running properly.
+  To see all of the Docker processes/containers that are actively running, use the following command:
+
+  .. code-block:: console
+
+    $ sudo docker ps
+
+  .. image:: /img/setup-azure/azure-connecting-to-virtual-machine2.png
+    :width: 600
+
+  If there are 9 (or 8 without https) services running under the name
```

Comment:
Does this go to a different number after something finishes? I remember discussing this.

Reply:
Yes, we discussed this. It was 8 earlier but now total 9 services run on the server (one is HTTPS , and one stopped with 7 other actively running containers). Hence, I have added the command (sudo docker ps -a) later which allows the user to look at all (both stopped and running) services on the server along with an image.

Reply:
A-ha, awesome!